packages/odyssey-react/src/components/Table/Table.test.tsx

```diff
@@ -39,7 +39,7 @@ describe("Table", () => {
       <Table caption={caption} title={tableTitle} />
     );
 
-    expect(getByRole('table', { name: caption })).toBe(true);
+    expect(getByRole('table', { name: caption })).toBeTruthy();
```

Comment:
consider an assertion here such as `.toBeVisible()` which is slightly more useful than asserting against the truthyness of `getByRole` as it will throw if no match is found.